frontend/src/hooks/api/getters/useImpactMetricsData/useImpactMetricsData.ts

```diff
@@ -10,6 +10,7 @@ export type ImpactMetricsSeries = {
     data: TimeSeriesData;
 };
 
+// TODO: use OpenAPI types
```

Comment:
Can we tag this with a flag or something so that we remember to fix this at some point? 

Something like this, just with the right flag name?
```suggestion
// TODO(impactMetrics): use OpenAPI types
```